MAUI/DataGrid/data-binding.md

```diff
@@ -0,0 +1,116 @@
+
+When you bind collection to the `ItemsSource` property of the `SfDataGrid`, then View will be created and maintains the operations on Data such as Sorting, Filtering, Insert, Delete, and Modification.
+
+>Note: The `SfDataGrid` creates different types of view derived from `ICollectionViewAdv` interface based on the `ItemsSource` property.
+
+### LiveDataUpdateMode
+
+During data manipulation operations and property changes, the `SfDataGrid` supports updating the view by using the `LiveDataUpdateMode` property which exists in the [SfDataGrid.View](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Data.ICollectionViewAdv.html#Syncfusion_Maui_Data_ICollectionViewAdv) class.
+
+<table>
+<tr>
+<th>LiveDataUpdateMode</th>
+<th>Description</th>
+</tr>
+<tr>
+<td>Default</td>
+<td>Data operations are not updated. </td>
+</tr>
+<tr>
+<td>AllowSummaryUpdate</td>
+<td>Summaries are updated during data manipulation change.</td>
+</tr>
+<tr>
+<td>AllowDataShaping</td>
+<td>Data operations like sorting, and filtering are updated during data manipulation change.
+</td>
+</tr>
+</table>
+
+{% tabs %}
+{% highlight c# %}
+this.sfDataGrid.View.LiveDataUpdateMode = LiveDataUpdateMode.Default;
+{% endhighlight %}
+{% endtabs %}
+
+### RecordPropertyChanged
+
+The [RecordPropertyChanged](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Data.ICollectionViewAdv.html#Syncfusion_Maui_Data_ICollectionViewAdv_RecordPropertyChanged) event is invoked when the value of `DataModel` property is changed, if `DataModel` implements the `INotifyPropertyChanged` interface. The event receives with two arguments namely `sender` that handles the `DataModel` and the [PropertyChangedEventArgs](https://docs.microsoft.com/en-us/dotnet/api/system.componentmodel.propertychangedeventargs?view=net-6.0) as argument. `PropertyChangedEventArgs` object contains the following property,
+
+* **PropertyName**: Defines the `PropertyName` of the changed value in the `DataModel`.
+
+### SourceCollectionChanged
+
+The [SourceCollectionChanged](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Data.ICollectionViewAdv.html#Syncfusion_Maui_Data_ICollectionViewAdv_SourceCollectionChanged) event is invoked when the `SourceCollection` is changed. For example, adding or removing the collection. The event receives two arguments namely `sender` that handles [GridQueryableCollectionViewWrapper](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.GridQueryableCollectionViewWrapper.html) object and the [NotifyCollectionChangedEventArgs](https://docs.microsoft.com/en-us/dotnet/api/system.collections.specialized.notifycollectionchangedeventargs?view=net-6.0) as argument. `NotifyCollectionChangedEventArgs` object contains the following properties,

```

Comment:
The [SourceCollectionChanged](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Data.ICollectionViewAdv.html#Syncfusion_Maui_Data_ICollectionViewAdv_SourceCollectionChanged) event is invoked when the `SourceCollection` is changed. For example, adding or removing the collection. The event receives two arguments namely `sender` that handles [GridQueryableCollectionViewWrapper](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.GridQueryableCollectionViewWrapper.html) object and the [NotifyCollectionChangedEventArgs](https://docs.microsoft.com/en-us/dotnet/api/system.collections.specialized.notifycollectionchangedeventargs?view=net-6.0) as argument. `NotifyCollectionChangedEventArgs` object contains the following properties,

Reply:
Modified.